scenariorun/src/main/openapi/scenariorun.yaml

```diff
@@ -786,6 +786,10 @@ components:
         sdkVersion:
           type: string
           description: the MAJOR.MINOR version used to build the solution solution
+        creationDate:
```

Comment:
From the look I took when adding similar fields to the modeling API, it seems other time point fields use a `<action>edAt` name pattern, should this one be consisent too and be `createdAt`?